### ISSUE TITLE
Numerically Stable PIP computation

### ISSUE THREAD

**Issue by quattro** [mancusolab/susiepca#13]
If `L` is large, computing PIPs can be numerically unstable. Let's switch to something like, 

```python
pip = 1 - jnp.exp(jnp.sum(jnp.log1p(-alpha), axis=-1))
```

**Comment by Dong555** [comment]
update completed

**Comment by quattro** [comment]
Should #13 be `axis=-1` or `axis=0` ?

**Comment by Dong555** [comment]
axis = 0 is correct, fixed!
